pom.xml

```diff
@@ -5,7 +5,7 @@
 	<groupId>com.vaadin.pekka</groupId>
 	<artifactId>resizablecsslayout-root</artifactId>
 	<packaging>pom</packaging>
-	<version>1.1.0</version>
+	<version>1.2.0</version>
```

Comment:
Changing Java version makes this a breaking change, thus I would go up to 2.0 even without any added features.

Reply:
Done